.maintain/k8s-resources/gitlab/values.yaml

```diff
@@ -30,7 +30,7 @@ gitlabUrl: https://gitlab.parity.io/
 #
 # runnerRegistrationToken: ""
 # simnetscripts
-runnerRegistrationToken: "LiKxi3TkzuAUidwMMizm"
+runnerRegistrationToken: "3d5gGCsqHXBLyxx8wUxf"
```

Comment:
this is something you should keep in secrets

Reply:
In order to register a runner in kubernetes, one needs to create a gitlab admin service account 
if this is a security concern I can remove it from the config and specify it as cli flag when I install the runner

Reply:
this token should be taken from API every time a new one, we have it implemented in ansible scripts for the runner deployment.

Reply:
I removed the token from the config value